### PR TITLE
bind port connections table for summarised data

### PR DESCRIPTION
**Pull request by yasin-cs-ko-ak**
This PR Contains 
- Bind Port Network Connection details for summarised data. 
- This will show when kubearmor log data has "SYS_BIND" under it's network data.
This is how sample output will look like.
```

Bind Port Connections
+----------+---------------------------+-----------------------------+-----------+--------------+-----------+--------+-------+------------------------------+
| PROTOCOL |          COMMAND          |         POD/SVC/IP          | BIND PORT | BIND ADDRESS | NAMESPACE | LABELS | COUNT |      LAST UPDATED TIME       |
+----------+---------------------------+-----------------------------+-----------+--------------+-----------+--------+-------+------------------------------+
| UNIX     | /usr/sbin/mysqld          | /var/run/mysqld/mysqld.sock |           |              |           |        | 2     | Tue Dec  6 08:16:13 UTC 2022 |
|          | /usr/lib/apt/methods/http |                             |           |              |           |        | 3     | Tue Dec  6 08:18:29 UTC 2022 |
|          | /bin/nc.traditional       |                             |           |              |           |        | 2     | Tue Dec  6 08:54:54 UTC 2022 |
+----------+---------------------------+-----------------------------+-----------+--------------+-----------+--------+-------+------------------------------+
```
if SYS_BIND doesn't happen it will show empty.

### CODE REVIEW

**Review by nyrahul — APPROVED** [review]

File: summary/table.go [hunk 1]
```diff
@@ -141,6 +143,28 @@ func DisplaySummaryOutput(resp *opb.Response, revDNSLookup bool, requestType str
 			WriteTable(SysNwHeader, outNwRowData)
 			fmt.Printf("\n")
 		}
+
+		if len(resp.BindConnection) > 0 {
+			fmt.Printf("\nBind Port connections\n")
```
Comment (nyrahul) [inline summary/table.go:148]:
```suggestion
			fmt.Printf("\nBind points\n")
```

**Review by nyrahul — CHANGES_REQUESTED** [review]

File: summary/table.go (hunk 1 above)
Comment (yasin-cs-ko-ak) [reply summary/table.go]: Renamed the title.

**Comment by yasin-cs-ko-ak** [comment]
The output:
```
Bind Points
+----------+------------------+-----------+--------------+-------+------------------------------+
| PROTOCOL |     COMMAND      | BIND PORT | BIND ADDRESS | COUNT |      LAST UPDATED TIME       |
+----------+------------------+-----------+--------------+-------+------------------------------+
| UNIX     | /usr/sbin/mysqld |           |              | 2     | Wed Dec  7 14:04:27 IST 2022 |
|          | /usr/sbin/mysqld | 3306      | ::           | 1     | Wed Dec  7 14:04:27 IST 2022 |
+----------+------------------+-----------+--------------+-------+------------------------------+
```

**Review by seswarrajan — APPROVED** [review]
LGTM

**Review by nyrahul — APPROVED** [review]

**Pull request merged**